src/api/googlesheetApi.js

```diff
@@ -26,24 +25,22 @@ function normalizeHeaders(element) {
 
 }
 
-function sheetIdFromPath(directory, path) {
-  if(!!path || path === ""){
-    return;
+function sheetIdFromPath(directory, path){
+  for (var i=0; i < directory.length; i++) {
+      if (directory[i].path === path) {
+          return directory[i].sheetId;
+      }
   }
-
-  const requestedSite = directory.find(function(site) {
-    return site["path"] === path;
-  });
-
-  if(!!requestedSite){
-    return;
-  }
-  return requestedSite["sheetId"];
 }
 
-export let getAllResources = new Promise(function(resolve, reject){
+export let getAllResources = (resourcePath) => new Promise(function(resolve, reject){
   getAllSites.then(sites => {
-    const resourceSheetId = envSheetId || sheetIdFromPath(sites, resourcePath) || revereSheetId
+    const resourceSheetId = sheetIdFromPath(sites, resourcePath) || envSheetId ||  revereSheetId;
+    alert(resourceSheetId);
+    if(resourceSheetId == null){
+      alert("Error: Unable to find resource '" + resourcePath + "'");
```

Comment:
I normally dont like using `alert` since its pretty jarring if a user were to mistype & end up here with a confusing error message. Im happy to leave this for now & made a new github issue.

Reply:
Forgive me - still a WIP! I removed some extraneous code and fixed some more broken behavior.